src/main/java/core/basesyntax/model/User.java

```diff
@@ -58,4 +64,13 @@
     public int hashCode() {
         return Objects.hash(login, password, age);
     }
+
+    @Override
+    public String toString() {
+        return "User{"
+                + "id=" + id
+                + ", login='" + login + '\''
+                + ", password='" + password + '\''
```

Comment:
Including the password in the `toString` method might expose sensitive information. Consider removing it from the string representation of the object.